ItemSwap.cs

```diff
@@ -184,11 +184,12 @@ public static void WriteNewItem(int ItemSlot, int NewItem, bool IsRepeatable, bo
                 WriteToROM(0xB499A6, (ushort)GetItemIndices[ItemSlot]);
                 WriteToROM(0xC72B5A, (ushort)GetItemIndices[ItemSlot]);
             };
-            if (NewItem == 14)
-            {
-                WriteToROM(0xB499B2, (ushort)GetItemIndices[ItemSlot]);
-                WriteToROM(0xC72B66, (ushort)GetItemIndices[ItemSlot]);
-            };
+            // Goron Race Bottle now rewards a regular Bottle, so this is unnecessary until a proper fix for Goron Dust is found.
```

Comment:
I would replace the Goron Race Bottle with the Gold Dust and have the new Bottle be a random item. This would make the logic more accurate.

Reply:
Currently there's only one check in the logics that require Gold Dust. That is the Gilded Sword check. So I updated that check in Casual and Glitched logic files. Users with User Logic will need to update just that check to use Gold Dust instead of Goron Race Bottle.

If we do the opposite, i.e. replace Goron Race Bottle with Gold Dust, and Mountain Village Grotto (Spring) with a Bottle, then there are far more changes required to the logics, and users with User Logic will need to go through and update a lot of logic. Then if/when this issue is fixed properly, they'll need to go back and change them all back.

In my opinion, the way I've done it is the cleanest way.

Reply:
Fixed the way you suggested. Updated logic.